--- R-4.2.0-orig/configure.ac	2022-04-14 15:02:01.000000000 -0700
+++ R-4.2.0/configure.ac	2022-04-27 16:20:40.418703334 -0700
@@ -882,7 +882,7 @@
   if test "${use_readline}" = no; then
     ## only need ncurses if libreadline is not statically linked against it
     unset ac_cv_lib_readline_rl_callback_read_char
-    AC_CHECK_LIB(ncurses, main, [],
+    AC_SEARCH_LIBS([initscr], [ncursesw curses], [],
                  AC_CHECK_LIB(tinfo, main, [],
                               AC_CHECK_LIB(termcap, main, [],
                                            AC_CHECK_LIB(termlib, main))))
