--- R-4.2.0-orig/src/library/tcltk/R/unix/zzz.R	2022-03-24 16:02:04.000000000 -0700
+++ R-4.2.0/src/library/tcltk/R/unix/zzz.R	2022-04-27 15:53:01.781405234 -0700
@@ -49,7 +49,7 @@
             ind <- grep("libtk[.0-9]+[.]dylib", out)
             if(length(ind)) {
                 this <- sub(" .*", "", sub("^\t", "", out[ind]))
-                if(!file.exists(this)) {
+                if(!file.exists(this) && !grepl("@rpath",this)) {
                     ## one issue here is that libtk built from unpatched
                     ## sources has wrong id, so we report what it is looking for
                     ## (/opt/R/arm64/lib:/usr/X11R6/lib/libtk8.6.dylib is wrong)
